src/audio/selector/selector.c

```diff
@@ -577,6 +577,8 @@ SOF_MODULE_INIT(selector, sys_comp_selector_init);
 static void build_config(struct comp_data *cd, struct module_config *cfg)
 {
 	enum sof_ipc_frame __sparse_cache frame_fmt, valid_fmt;
+	const struct sof_selector_ipc4_config sel_cfg  = cd->sel_ipc4_cfg;
```

Comment:
nitpick: one extra space "cfg  ="

Reply:
The CI TEST previously halt due to CI issue, I updated the code, and re-trigger CI test with new push, fortunately the approvals are not dismissed.